x-pack/plugins/triggers_actions_ui/public/application/components/builtin_alert_types/threshold/expression.tsx

```diff
@@ -283,7 +283,7 @@ export const IndexThresholdAlertTypeExpression: React.FunctionComponent<IndexThr
   const firstSetOfSteps = [
     {
       title: i18n.translate('xpack.triggersActionsUI.sections.alertAdd.selectIndex', {
-        defaultMessage: 'Select an index.',
+        defaultMessage: 'Select an index',
```

Comment:
Changing translations should also remove the keys from the following files. This will cause the ids to be missing and require them re-translate:
- x-pack/plugins/translations/translations/ja-JP.json
- x-pack/plugins/translations/translations/zh-CN.json

Two ids in this file:
- xpack.triggersActionsUI.sections.alertAdd.selectIndex
- xpack.triggersActionsUI.sections.alertAdd.conditionPrompt

And possibly more:
- xpack.triggersActionsUI.sections.alertsList.bulkActionPopover.muteAllTitle
- xpack.triggersActionsUI.sections.alertsList.bulkActionPopover.unmuteAllTitle
- xpack.triggersActionsUI.sections.alertsList.bulkActionPopover.enableAllTitle
- xpack.triggersActionsUI.sections.alertsList.bulkActionPopover.disableAllTitle
- xpack.triggersActionsUI.sections.alertsList.bulkActionPopover.deleteAllTitle

Reply:
I updated translations for ids `xpack.triggersActionsUI.sections.alertAdd.selectIndex`
and `xpack.triggersActionsUI.sections.alertAdd.conditionPrompt`. Since I only removed a `.` it was easy to know what to delete in the japanese and mandarin translations.